www/source/blog/2017-06-14-Habitat-Issue-Tracker.html.md

```diff
@@ -0,0 +1,74 @@
+
+Lots of new features might require community input. So, we suggest that the contributor whom submits the issue make sure to post about it in community channels to drum up user input. Contributors should also feel comfortable using the `@core-maintainers` slack notifier to notify the maintainers that the RFC has been opened.
+
+In the case of an RFC being opened. The core maintainers may request separate issues to be created for various aspects of the feature (dependent wholly on the size of the incoming work) which will be linked back to the original RFC. In the case that the RFC is a manageable chunk of work for an individual the issue will get sorted to Accepted Minor or Accepted Major milestones, the core engineering project board and the RFC prefix will be removed.
+
+## Merging Code
+* Open a Pull Request that includes a Developer Certificate of Origin-signed (DCO-signed) commit (anyone).
+* Code is reviewed by a Maintainer, Lieutenant, or Project Lead. Approval is indiciated by a thumbs up in a comment on the PR.
+* Merged after a thumbs up vote by at least one Maintainer in the component(s) affected by your patch (Area Tag). The merge will be initiated by approved maintainer
+
+## Pull Request Review and Merge Automation
+Habitat uses several bots to automate the review and merging of pull requests. Messages to and from the bots are brokered via the account @thesentinels. First, we use Facebook's mention bot to identify potential reviewers for a pull request based on the blame information in the relevant diff. @thesentinels can also receive incoming commands from reviewers to approve PRs. The bot will first fire pre-commit checks across your PR to ensure that a DCO signature is provided and that the code doesn't have easily corrected issues. Once the PR has approval from a maintainer they will issue an `@thesentinels approve` command which will be routed to a bot that will automatically merge the patch.
+
+Any Maintainer may downvote on a PR, which increases the requirement for a patch to be merged to an absolute majority of Maintainers for the affected component(s), unless that Maintainer later changes their vote.
+
+## Patch Appeals Process
+
+There may be cases where someone wishes to appeal a Maintainer decision. In this event, the "chain of command" for the appeals process is as follows.
+
+    In the event that the actions of a Maintainer are to be appealed, the appeal should be directed to the Lieutenant for that component. As stated above, a Lieutenant retains veto power for the component(s) for which they are responsible.
```

Comment:
These two blocks are being formatted as code, which is causing them to be rendered on a single side-scrolling line. Blockquotes (`>`) might make them a little more readable.